### ISSUE TITLE
Unable to fix a ControlStructures.BlockControlStructureSpacing violation

### ISSUE THREAD

**Issue by morozov** [slevomat/coding-standard#936]
```sh
$ composer show | grep -P 'coding-standard|php_codesniffer'

doctrine/coding-standard                       7.0.2              The Doctrine Coding Sta...
slevomat/coding-standard                       dev-master 3c557d5 Slevomat Coding Standar...
squizlabs/php_codesniffer                      3.5.4              PHP_CodeSniffer tokeniz...
```

Consider the following code example:

```php
<?php

declare(strict_types=1);

// begin block
if (true) {
    echo 1;
} // end block

// comment
if (true) {
    echo 2;
}
```

The following violation is reported:
```
------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 6 | ERROR | [x] Expected 1 lines after "if", found 2.
   |       |     (SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------
```
An attempt to auto-fix the file fails:
```
PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
test.php                                              FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------
```
The failure to fix this little one prevents PHPCBF from fixing the rest of the violations in the actual file that contains this violation.

**Comment by kukulich** [comment]
I really hate the inline comments after code :)

**Comment by morozov** [comment]
I do too. They come from a legacy codebase on which I'm testing the coding standard. Prior to getting such issues fixed, I remove them manually since they are useless for the most part.

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/653364df41617768636176d6c68244cbcac773c7
